### ISSUE TITLE
Correct filesystem module

### ISSUE THREAD

**Issue by sirdanielk** [RPCS3/rpcs3#3483]
With newest build Wolfenstein NPUB90248 gets into loop with this 
```
W {PPU[0x1000007] Thread (fios mediathread) [0x017c5ec0]} sys_fs: 0xc0000006: 0x20, 0x10, 0x18, “/dev_hdd0/game/NPUB90248/USRDIR/baseps3/app_home/c:/temp/wolf2/baseps3/scripts/scripts/lib/defs.script”, 0x0, 0x80010003, 0x0
W {PPU[0x1000007] Thread (fios mediathread) [0x017c7090]} sys_fs: sys_fs_open(path=“/dev_hdd0/game/NPUB90248/USRDIR/baseps3/app_home/c:/temp/wolf2/baseps3/scripts/scripts/lib/defs.script”, flags=0, fd=*0xd00ca3d0, mode=0, arg=*0x0, size=0x0)
E {PPU[0x1000007] Thread (fios mediathread) [0x017c7090]} sys_fs: sys_fs_open(): unknown error Invalid arguments
E {PPU[0x1000007] Thread (fios mediathread) [0x017c7090]} 'sys_fs_open' failed with 0x8001002b : CELL_EIO, “/dev_hdd0/game/NPUB90248/USRDIR/baseps3/app_home/c:/temp/wolf2/baseps3/scripts/scripts/lib/defs.script” [1]
W {PPU[0x1000007] Thread (fios mediathread) [0x017c5ec0]} sys_fs: 0xc0000006: 0x20, 0x10, 0x18, “/”, 0x0, 0x80010003, 0x0
W {PPU[0x1000007] Thread (fios mediathread) [0x017c6ee4]} sys_fs: sys_fs_stat(path=“/”, sb=*0xd00ca2b8)
W {PPU[0x1000007] Thread (fios mediathread) [0x017c6380]} sys_fs: sys_fs_fstat(fd=6, sb=*0xd00ca390)
W {PPU[0x1000007] Thread (fios mediathread) [0x017c6380]} sys_fs: sys_fs_fstat(fd=9, sb=*0xd00ca390)
W {PPU[0x1000007] Thread (fios mediathread) [0x017c5ec0]} sys_fs: 0xc0000006: 0x20, 0x10, 0x18, “/dev_hdd0/game/NPUB90248/USRDIR/baseps3/app_home/c:/temp/wolf2/baseps3/scripts/scripts/lib/defs.script”, 0x0, 0x80010003, 0x0
```
Additionally, CoJ BiB Demo need to its files stored in dev_bdvd folder. I placed files, changed config file accordingly, but game still does not detect files. This old trick doesnt work for quite long.
Maybe its because double slash? 
```

E {PPU[0x1000000] Thread (main_thread) [0x014238a8]} 'sys_fs_open' failed with 0x8001003a : CELL_ENOTMOUNTED, “/dev_bdvd/PS3_GAME/USRDIR/CoJ2/Data0.pak” [1]
W {PPU[0x1000000] Thread (main_thread) [0x01ff4218]} sys_lwmutex: _sys_lwmutex_destroy(lwmutex_id=0x95007400)
W {PPU[0x1000000] Thread (main_thread) [0x0209aec0]} sys_fs: 0xc0000006: 0x20, 0x10, 0x18, “/dev_bdvd/PS3_GAME/USRDIR/Out/cache/filesystem/cell/Cell_5343ac85.fsc”, 0x0, 0x80010003, 0x0
W {PPU[0x1000000] Thread (main_thread) [0x0209bee4]} sys_fs: sys_fs_stat(path=“/dev_bdvd/PS3_GAME/USRDIR/Out/cache/filesystem/cell/Cell_5343ac85.fsc”, sb=*0xd01000b8)
W {PPU[0x1000000] Thread (main_thread) [0x0209bee4]} vfs::get(): device not found: /dev_bdvd/PS3_GAME/USRDIR/Out/cache/filesystem/cell/Cell_5343ac85.fsc
```
```
W {PPU[0x1000000] Thread (main_thread) [0x0202bd4c]} sys_spu: sys_spu_thread_group_connect_event(id=0x1, eq=0x8d00d400, et=2)
U {PPU[0x1000000] Thread (main_thread) [0x01ffef0c]} sys_prx TODO: _sys_prx_get_module_id_by_name(name=“cellLibprof”, flags=0, pOpt=*0x0)
E {PPU[0x1000000] Thread (main_thread) [0x01ffef0c]} '_sys_prx_get_module_id_by_name' failed with 0x8001112e : CELL_PRX_ERROR_UNKNOWN_MODULE [1]
W {PPU[0x1000008] Thread (Engine SPURS SpursHdlr0) [0x02027378]} sys_spu: sys_spu_thread_group_start(id=0x1)
W {PPU[0x1000000] Thread (main_thread) [0x019b92e0]} sceNp: sceNpDrmIsAvailable2(k_licensee=*0xd010034c, drm_path=“/dev_bdvd/PS3_GAME/USRDIR//ps3/engine_jobs.sprx”)
W {PPU[0x1000000] Thread (main_thread) [0x019b92e0]} vfs::get(): device not found: /dev_bdvd/PS3_GAME/USRDIR//ps3/engine_jobs.sprx
S {SPU[0x2000001] Thread (Engine SPURS CellSpursKernel1)} SPU: SPU Database initialized...
W {PPU[0x1000000] Thread (main_thread) [0x019b92e0]} sceNp: npDrmIsAvailable(): '/dev_bdvd/PS3_GAME/USRDIR//ps3/engine_jobs.sprx' not found
W {PPU[0x1000000] Thread (main_thread) [0x01fff9ac]} sys_prx: _sys_prx_load_module(path=“/dev_bdvd/PS3_GAME/USRDIR//ps3/engine_jobs.sprx”, flags=0x0, pOpt=*0x0)
W {PPU[0x1000000] Thread (main_thread) [0x01fff9ac]} vfs::get(): device not found: /dev_bdvd/PS3_GAME/USRDIR//ps3/engine_jobs.sprx
S {SPU[0x2000001] Thread (Engine SPURS CellSpursKernel1)} SPU: SPU Recompiler (ASMJIT) created...
E {PPU[0x1000000] Thread (main_thread) [0x01fff9ac]} '_sys_prx_load_module' failed with 0x8001113d : CELL_PRX_ERROR_ILLEGAL_LIBRARY [1]
W {PPU[0x1000000] Thread (main_thread) [0x01ff3f74]} sys_lwmutex: _sys_lwmutex_create(lwmutex_id=*0xd00fffb0, protocol=0x2, control=*0x19bd168, arg4=0x80000001, name=0x5f6c763270726900, arg6=0x0)
W {PPU[0x1000000] Thread (main_thread) [0x01fff9ac]} sys_prx: _sys_prx_load_module(path=“/dev_flash/sys/external/libfiber.sprx”, flags=0x0, pOpt=*0x0)
```

**Comment by kamer1337** [comment]
try #3467

**Comment by sirdanielk** [comment]
Unfortunately same situation. To make sure i created new config file.

**Comment by AniLeo** [comment]
Fixed by #3575
